src/org/dhsdev/flowerknight/gl/Renderable.java

```diff
@@ -0,0 +1,22 @@
+package org.dhsdev.flowerknight.gl;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a renderable object
+ * @author cameron
+ */
+public interface Renderable {
+    public static ArrayList<Renderable> renderables = new ArrayList<Renderable>();
```

Comment:
use `List<Renderable> renderables = new ArrayList<>();`

Reply:
also why public?

Reply:
We can make a pr to fix that

Reply:
> also why public?

Because it is accessed in the main class.